### DISCUSSION TITLE
Recording and storing into memory as MP3 with VS1053

### DISCUSSION THREAD

**Discussion by mihai-en** [baldram/ESP_VS1053_Library#113] [Q&A]
I was able to play an MP3 from memory and network using this library using an Olimex ESP32 POE ISO and a Olimex MOD-MP3-X-LITE  ( based on  VS1053 -  version without ST additional microcontroller ), both connected via UEXT connector.  I have attached part of the source code. 
[mp3_Test (copy).txt](https://github.com/user-attachments/files/16825906/mp3_Test.copy.txt)  maybe is usefull for other. 

Any idea if is possible to record an mp3 and stream the data in real time to memory / network ?  I was not able to do that currently.

**Comment by baldram** [comment]
I have never tried it. However, it seems that it is technically possible with VS1053. 
For example, see this thread: https://forum.arduino.cc/t/how-to-record-voice-using-vs1053-and-esp32-arduino-ide/1169716
